docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -89,3 +89,18 @@ We are gradually rolling out our new browse dashboards user interface. With this
 To learn more, refer to the following video demo.
 
 {{< video-embed src="/media/docs/grafana/2023-09-11-New-Browse-Dashboards-Enablement-Video.mp4" >}}
+
+## Authentication and authorization
+
+### No basic role
```

Comment:
In the on-prem What's new, we have sections, but for now in the Cloud version, we don't. 
```suggestion
## No basic role
```